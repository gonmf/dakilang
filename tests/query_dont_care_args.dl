
type('fruit', 'apple', 120).
type('fruit', 'banana', 90).

type('fruit', 'apple', _)?

type('fruit', Name, _)?

type(_, _, _)?

