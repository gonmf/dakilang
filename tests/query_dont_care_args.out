>
> type('fruit', 'apple', 120).
> type('fruit', 'banana', 90).
>
> type('fruit', 'apple', _)?
Yes

>
> type('fruit', Name, _)?
Name = 'apple'

Name = 'banana'

>
> type(_, _, _)?
Yes

>
